runtime/tests/staking_reward.rs

```diff
@@ -211,19 +211,13 @@ fn current_era_transaction_rewards_storage_update_works() {
 }
 
 #[test]
-fn elected_validators_receive_equal_transaction_fee_reward() {
+fn elected_validators_receive_transaction_fee_according_to_authorship_points() {
 	// Make some txs
 	// Start a new era to payout last eras validators
 	// Check payouts happen as expected and total issuance is maintained
 	let validators = make_authority_keys(6);
 	let initial_balance = 100_000_000 * DOLLARS;
 	let staked_amount = initial_balance / validators.len() as Balance;
-	let transfer_amount = 50;
```

Comment:
it's important we still test actual transaction fees are recorded and distributed here, maybe you forgot to add this stuff back into the test

Reply:
We do test applying an extrinsic and its effect on the transaction fee pot in current_era_transaction_rewards_storage_update_works. Here the challenge that I had was that we do need to start a block for the amortised payments to happen. However, starting a new block affects the authorship reward points and the distribution of the next rewards. To keep the whole test simple and solid, I just allowed the authorship to happen in a controlled way only on Executive::initialize_block.

Reply:
this was testing that transaction fees from a block are paid out to stakers. It doesn't make sense to remove this, you need to make another test

Reply:
I will add another test for checking the transaction fee is paid to the stakers and the treasury gets its cut.